services/frontend-service/src/ui/Pages/Home/Home.tsx

```diff
@@ -26,8 +33,18 @@ export const Home: React.FC = () => {
     const searchedApp = useSearchedApplications(filteredApps, appNameParam);
 
     const apps = Object.values(searchedApp);
+
+    const { app, version } = useReleaseDialog(({ app, version }) => ({ app, version }));
+
+    const envs = useCurrentlyDeployedAt(app, version);
+    let releaseInfo = useReleaseInfo(app, version);
```

Comment:
```suggestion
    let releaseInfo = useReleaseInfo(app, version);
```
please don't use let, only const